<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="stack_login">
     <widget class="QLineEdit" name="input_login_id">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>로그인</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_login_pw">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>비밀번호</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>로그인</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login_to_signup">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stack_signup">
     <widget class="QLineEdit" name="input_signup_id">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>130</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>아이디</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_signup_pw">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>190</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>비밀번호</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_signup_pwck">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>250</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>비밀번호 확인</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_signup_name">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>310</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>이름</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_signup_phone">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>370</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>전화번호</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_signup_id">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_signup_pw">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>220</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_signup_pwck">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>280</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>아이디</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>비밀번호</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>비밀번호 확인</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>이름</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>370</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>전화번호</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_signup_to_login">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_signup">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>480</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stack_main">
     <widget class="QPushButton" name="btn_logout">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>로그아웃</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>521</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>지역 선택</string>
      </property>
      <widget class="QPushButton" name="map_view_btn">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>40</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>지도 표시</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>261</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>시</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>읍 · 면 · 동</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="location1">
          <item>
           <property name="text">
            <string>제주시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>서귀포시</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="location2"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="send_btn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>700</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>조회하기</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="mapbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>521</width>
        <height>531</height>
       </rect>
      </property>
      <property name="title">
       <string>지도를 움직여 상세 지역을 선택해주세요.</string>
      </property>
      <widget class="QWebEngineView" name="map" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>500</width>
         <height>500</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="url" stdset="0">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="stack_info">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>추천 업종</string>
      </property>
      <widget class="QLabel" name="la">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1위 : 추천 프랜차이즈 or 개인사업</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>721</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>주위의 평점이 높은 장소</string>
      </property>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>이름 / 카테고리(비즈니스) / 위치 좌표</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="list_best">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>721</width>
         <height>181</height>
        </rect>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>이름</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>분류</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>주소</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>평점</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>1041</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>반경 1Km 이내의 업종</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>25</y>
         <width>1031</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="result_category"/>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_info_to_main">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
